README.md

```diff
@@ -244,4 +244,5 @@ registerKerasImageUDF("my_keras_inception_udf", InceptionV3(weights="imagenet"),
 ### Estimator
 
 ## Releases:
-* 0.1.0 initial release
+* 0.1.0 Alpha release
```

Comment:
Just append at the bottom. 
```0.2.0 release: Keras estimator,...
```
It should be the same as the release notes when cutting the github branch for the release.